keps/sig-api-machinery/20190228-priority-and-fairness.md

```diff
@@ -1047,7 +1058,273 @@ such objects would test that the workload does not crowd out the
 garbage collector.
 
 
+### Observed Requests
 
+To provide data about requests to use in designing the configuration,
+here are some observations from a running system late in the
+release 1.16 development cycle.  These are extracts from the
+kube-apiserver log file, with linebreaks and indentation added for
+readability.
+
+These are from debug log statements in an early revision of the
+implementation of this KEP, because that is the only easy source of
+the relevant request details.  Do not pay attention to the
+classification results evident in these log messages, they are not
+what is important here; what is important is the `requestInfo` and the
```

Comment:
Thanks for calling out what you're trying to represent.  Because these are simple `%v` statements, you can only interpret the values if you know exactly which level of code they came from.  This makes sense for debugging purposes (you know which version you are), but doesn't make as much sense for a doc like this that needs to span multiple releases.

Perhaps you could render them as json or yaml or something that is self-identifying for fields in the future. Future us will want more detail than `true` for instance.